torch/autograd/function.py

```diff
@@ -48,6 +48,7 @@ def save_for_backward(self, *tensors: torch.Tensor):
         See :ref:`extending-autograd` for more details on how to use this method.
 
         Example::
+            >>> # xdoctest: +REQUIRES(env:TORCH_DOCTEST_AUTOGRAD)
```

Comment:
So how exactly do the REQUIRES annotations work, do we have to remember to put then on all the autograd related doctests?